package.json

```diff
@@ -6,16 +6,15 @@
   "scripts": {
     "build": "npm run build:css",
     "build:css": "node-sass --output-style=expanded --output=server/public/build/stylesheets client/sass",
-    "pretest": "npm run test:lint",
-    "start:watch": "nodemon --inspect=0.0.0.0 --ext js,njk --legacy-watch index.js",
-    "test": "npm run test:unit",
+    "test": "npm run test:lint && npm run test:unit",
```

Comment:
why has test reverted to using `&&` to run two commands, from the previous `pre-test` approach?

Reply:
To be consistent with the other services.  The test image doesn't run pretest so linting test wouldn't be run.

Reply:
Just asking the question, really. We don't document locking down the file system as readonly and other kubernetes best practices, so this wouldn't need explicitly mentioning either.